cob_bringup/robots/raw3-3.xml

```diff
@@ -3,6 +3,7 @@
 
 	<!-- args -->
 	<arg name="robot" value="raw3-3"/>
+	<arg name="include_non_released_packages" default="false"/>
```

Comment:
Can we use a more related arg name, e.g. `<arg name="use_rplidar" default="false"/>`?